### PR TITLE
PP-13135: Edit email collection mode cypress test

### PR DESCRIPTION
**Pull request by oswaldquek**

### CODE REVIEW

File: app/views/simplified-account/settings/email-notifications/edit-email-collection-mode.njk [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                checked: emailCollectionMode === 'OFF',
+                hint: {
+                  text: 'Users do not have the option to enter an email address – they will not receive notifications'
+                },
+                disabled: isReadyOnly
+            }
+          ]
+        })
+      }}
+      {% if not isReadyOnly %}
```
Comment (nlsteers) [inline app/views/simplified-account/settings/email-notifications/edit-email-collection-mode.njk:59]:
```suggestion
      {% if not isReadOnly %}
```

File: app/views/simplified-account/settings/email-notifications/edit-email-collection-mode.njk [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+{% endblock %}
+
+{% block settingsContent %}
+
+    {{ govukBackLink({
+      text: "Back",
+      href: backLink
+    }) }}
+
+    <form id="collection-email-form" method="post" novalidate>
```
Comment (nlsteers) [inline app/views/simplified-account/settings/email-notifications/edit-email-collection-mode.njk:14]:
```suggestion
    <form id="edit-email-collection-mode" method="post" novalidate>
```

File: app/views/simplified-account/settings/email-notifications/edit-email-collection-mode.njk [hunk 3]
```diff
@@ -0,0 +1,69 @@
+{% extends "../settings-layout.njk" %}
+
+{% block settingsPageTitle %}
+  Email notifications - {{currentService.name}} {{currentGatewayAccount.full_type}} - GOV.UK Pay
```
Comment (nlsteers) [inline app/views/simplified-account/settings/email-notifications/edit-email-collection-mode.njk:4]: see the [settings layout](https://github.com/alphagov/pay-selfservice/blob/db3fe0987cbc47ae5bfeac1e22f90f09ebe87145/app/views/simplified-account/settings/settings-layout.njk#L5) for how to use settingsPageTitle, some parts of the title are included for you (like `- GOV.UK Pay`)

File: app/views/simplified-account/settings/email-notifications/edit-email-collection-mode.njk [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+          },
+          items: [
+            {
+                value: emailCollectionModes.mandatory,
+                text: 'Yes – as a mandatory field',
+                checked: emailCollectionMode === 'MANDATORY',
+                hint: {
+                  text: 'Users must enter an email address to complete a payment – they can receive notifications'
+                },
+                disabled: isReadyOnly
```
Comment (nlsteers) [inline app/views/simplified-account/settings/email-notifications/edit-email-collection-mode.njk:36]:
```suggestion
                disabled: isReadOnly
```

File: app/views/simplified-account/settings/email-notifications/edit-email-collection-mode.njk [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+            },
+            {
+                value: emailCollectionModes.optional,
+                text: 'Yes – as an optional field',
+                checked: emailCollectionMode === 'OPTIONAL',
+                hint: {
+                  text: 'Users can choose to enter an email address – if they do, they can receive notifications'
+                },
+                disabled: isReadyOnly
```
Comment (nlsteers) [inline app/views/simplified-account/settings/email-notifications/edit-email-collection-mode.njk:45]:
```suggestion
                disabled: isReadOnly
```

File: app/views/simplified-account/settings/email-notifications/edit-email-collection-mode.njk [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+            },
+            {
+                value: emailCollectionModes.no,
+                text: 'No',
+                checked: emailCollectionMode === 'OFF',
+                hint: {
+                  text: 'Users do not have the option to enter an email address – they will not receive notifications'
+                },
+                disabled: isReadyOnly
```
Comment (nlsteers) [inline app/views/simplified-account/settings/email-notifications/edit-email-collection-mode.njk:54]:
```suggestion
                disabled: isReadOnly
```

**Review by nlsteers — CHANGES_REQUESTED** [review]
Looks good, few 💬

File: app/views/simplified-account/settings/email-notifications/edit-email-collection-mode.njk (hunk 1 above)
Comment (oswaldquek) [reply app/views/simplified-account/settings/email-notifications/edit-email-collection-mode.njk]: This was due to bad merge and should have been here 😩

**Pull request merged**